node/rust-client/src/lib.rs

```diff
@@ -415,6 +415,22 @@ pub fn create_leaked_double(float: f64) -> [u32; 2] {
     split_pointer(pointer)
 }
 
+/// Creates an open telemetry span with the given name and returns a pointer to the span
+#[napi(ts_return_type = "[number, number]")]
+pub fn create_leaked_otel_span(name: String) -> [u32; 2] {
+    let span = GlideOpenTelemetry::new_span(&name);
+    let s = Arc::into_raw(Arc::new(span)) as *mut GlideSpan;
+    split_pointer(s)
+}
+
+#[napi]
+pub fn drop_otel_span(span_ptr: BigInt) {
+    let span_ptr = span_ptr.get_u64().1;
```

Comment:
to be on the safe side, lets check that we haven't sent any garbage values (e.g., -1) from node and if we do log it.
something like-
```
let (is_negative, span_ptr) = span_ptr.get_u64();
if is_negative {
    log(Level::Error, "OpenTelemetry", "Failed to drop span: received a negative pointer value (expected non-negative)");}
else if span_ptr != 0 {
...
}